.github/workflows/build.yml

```diff
@@ -181,6 +186,49 @@ jobs:
           ${{ matrix.os }}-${{ matrix.compiler_cache }}-${{ github.head_ref }}
           ${{ matrix.os }}-${{ matrix.compiler_cache }}
 
+    - name: "Generate package filename"
```

Comment:
Please cherry pick this to a different branch. This needs a bigger discussion apart from the rest of this PR. I do not want this PR held back over this.

Reply:
#3388